exec/cnex/cell.c

```diff
@@ -283,14 +323,34 @@ TString *cell_toString(Cell *c)
             r = string_appendCString(r, (c->boolean ? "TRUE" : "FALSE"));
             break;
         case cDictionary:
+        {
+            size_t x;
+            r = string_appendCString(r, "{");
+            Cell *keys = dictionary_getKeys(c->dictionary);
+            for (x = 0; x < keys->array->size; x++) {
+                if (r->length > 1) {
+                    r = string_appendCString(r, ", ");
+                }
+                r = string_appendCString(r, "\"");
+                r = string_appendString(r, keys->array->data[x].string);
```

Comment:
Make a note that you need to backslash-quote this string properly at some point.

Reply:
Noted.